src/components/molecules/NavSearchBar/NavSearchResult.scss

```diff
@@ -0,0 +1,38 @@
+@import "scss/constants.scss";
+
+.nav-dropdown--search-result {
+  cursor: pointer;
+  padding: 8px 12px;
+  font-size: 0.9rem;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  align-content: space-between;
+
+  .nav-dropdown--search-result-image {
+    flex-shrink: 0;
+    height: 32px;
+    width: 32px;
+    margin-right: 10px;
+    border-radius: $border-radius--small;
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-color: $black;
+  }
+
+  .nav-dropdown--search-result-avatar {
+    height: 26px;
+    width: 26px;
+    border-radius: 50%;
+    background-size: cover;
+  }
```

Comment:
Why are the images and the avatars different sizes/etc?

Are we using one of our common components for rendering the avatars? If not, can we? (eg. `UserAvatar`).

Reply:
using `UserAvatar`: https://github.com/sparkletown/sparkle/pull/1328/commits/eb24607d724c23e94c3d4d1a40d7df37f6d9489b